### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#462] [Esami & Esoneri M-Z]
## Traccia

<img width="845" alt="image" src="https://github.com/user-attachments/assets/8cef802f-66e9-4340-8f96-e6b04c021c0c">

**Comment by rimaout** [comment]
<img width="923" alt="image" src="https://github.com/user-attachments/assets/1dab7ec1-651a-469f-bd28-3647aa3f5262">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/3dcea4cd-d66f-44e8-8e0b-f78e6c267ee0)
